specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,1026 @@
+
+        // Unique identifier used to replace a notification within a group.
+        String Tag;
+
+        // Unique identifier for a toast group in the app
+        String Group;
+
+        // A unique identifier for the Toast generated by the platform.
+        UInt32 ToastId;
+
+        // The notification Xml Payload
+        Windows.Data.Xml.Dom.XmlDocument Payload{ get; };
+
+        // Gets or sets additional information about the toast progress.
+        ToastProgressData ProgressData;
+
+        // Gets or sets the time after which a toast notification should not be displayed.
+        Windows.Foundation.DateTime ExpirationTime;
+
+        // Indicates whether the toast will remain in the Action Center after a reboot.
+        Boolean ExpiresOnReboot;
+
+        // Gets or sets the priority for a Toast.
+        // Hints on how and at what urgency level a notification should be presented to the user (whether to wake up the screen, etc).
+        ToastPriority Priority;
+
+        // Gets or sets whether a toast's pop-up UI is displayed on the user's screen.
+        Boolean SuppressDisplay;
+    };
+
+    [experimental]
+    // Supports Toast related operations for all prior displayed Toasts in Action Centre
+    runtimeclass ToastNotificationHistory
+    {
+        // Removes a specific toast with a specific toastIdentifier from Action Centre
+        Windows.Foundation.IAsyncAction RemoveWithIdentiferAsync(UInt32 toastIdentifier);
+
+        // Removes a toast having a specific tag
+        Windows.Foundation.IAsyncAction RemoveWithTagAsync(String tag);
+
+        // Removes a toast having a specific tag and group
+        Windows.Foundation.IAsyncAction RemoveWithTagGroupAsync(String tag, String group);
+
+        // Remove all toasts for a specific group
+        Windows.Foundation.IAsyncAction RemoveGroupAsync(String group);
+
+        // Removes all the toasts for the App from Action Centre
+        Windows.Foundation.IAsyncAction RemoveAllAsync();
+
+        // Gets all the toasts for the App from Action Centre
+        Windows.Foundation.IAsyncOperation<Windows.Foundation.Collections.IVector<ToastNotification> > GetAllAsync();
+    };
+
+    [experimental]
+    // The manager class which encompasses all Toast API Functionality
+    runtimeclass ToastNotificationManager
+    {
+        // Gets a Default instance of a ToastNotificationManager
+        static ToastNotificationManager Default{ get; };
+
+        // Register an activator using an ActivationInfo context and caches the token for unregister
+        void RegisterActivator(ToastActivationInfo details);
+
+        // Unregisters the activator and removes the cached Registration token.
+        void UnregisterActivator();
+
+        // Event handler for Toast Activations
+        event Windows.Foundation.EventHandler<ToastActivatedEventArgs> ToastActivated;
+
+        // Displays the Toast in Action Centre
```

Comment:
nit, Action Center to be consistent with the rest of the docs using en-us spelling